client/src/components/TripImageList/TripImageList.js

```diff
@@ -37,21 +37,21 @@ const useStyles = makeStyles({
   },
 });
 
-const TripImageList = ({ images, onRemove }) => {
+const TripImageList = ({ images, onRemove, isOwner }) => {
   const classes = useStyles();
 
   return (
     <Box className={classes.imageListContainer}>
       {images.map((img, i) => {
         return (
           <Card className={classes.cardContainer} key={i}>
-            <IconButton
+            {isOwner && <IconButton
               className={classes.icon}
               aria-label='remove picture'
               component='span'
               onClick={() => onRemove(img)}>
```

Comment:
could you please change this to: `onClick={isOwner ? () => onRemove(img) : null}>`